simtools/corsika/corsika_histograms_visualize.py

```diff
@@ -268,7 +268,13 @@ def _kernel_plot_1D_photons(histograms_instance, property_name, log_y=True):
             )
         else:
             ax.set_xlabel(f"{histograms_instance._dict_1D_distributions[property_name]['edges']} ")
-        ax.set_ylabel("Counts")
+        if property_name == "density":
+            ax.set_ylabel(
+                f"Density ("
+                f"{histograms_instance._dict_1D_distributions[property_name]['edges unit']}$^{-2}$)"
```

Comment:
I think the superscript did not work:

<img width="62" alt="Screenshot 2023-10-16 at 14 46 48" src="https://github.com/gammasim/simtools/assets/14001688/29fcfd7f-91a8-4dcd-a9d1-29d156397dea">